Triangulation_3/include/CGAL/Regular_triangulation_3.h

```diff
@@ -1711,7 +1711,7 @@ nearest_power_vertex(const Bare_point& p, Cell_handle start) const
   while(true)
   {
     Vertex_handle tmp = nearest;
-    adjacent_vertices(nearest, std::back_inserter(vs));
+    adjacent_vertices_threadsafe(nearest, std::back_inserter(vs));
```

Comment:
To fix the Travis error:
```suggestion
    this->adjacent_vertices_threadsafe(nearest, std::back_inserter(vs));
```
`adjacent_vertices_threadsafe` is implemented in `Triangulation_3` a dependent base class (dependent on template parameters).